src/ray/common/ray_config_def.h

```diff
@@ -461,17 +461,19 @@ RAY_CONFIG(int64_t, metrics_report_batch_size, 100)
 /// The interval duration for which task state events will be reported to GCS.
 /// The reported data should only be used for observability.
 /// Setting the value to 0 disables the task event recording and reporting.
-RAY_CONFIG(int64_t, task_events_report_interval_ms, 0)
+RAY_CONFIG(int64_t, task_events_report_interval_ms, 1000)
```

Comment:
This turns the backend on by default - or it could be made into another independent PR.

Reply:
I think it's better we turn this on by default asap (so that we can have more time to fix by 2.3 release). cc @ericl in case you disagree.

Reply:
Oh actually, we will need to turn the feature on with this change, so that profiling timeline events will be available. 

We could run nightly on this PR to get a baseline performance number.